django_webapps/scripts/db_population/populate_teachers.py

```diff
@@ -216,6 +217,17 @@ def _create_age_ranges(self, is_teaching_children: bool) -> QuerySetAny[AgeRange
     def _update_metadata(self, new_id: int, old_id: int, name: str) -> None:
         self._metadata[self.entity_name].append({"new_id": new_id, "old_id": old_id, "name": name})
 
+    def _create_or_get_personal_info(self, entity_data: TeacherData) -> None | PersonalInfo:
+        already_existing_one = PersonalInfo.objects.filter(
+            first_name=entity_data.first_name,
+            last_name=entity_data.last_name,
+            email=entity_data.email,
+        ).first()
+        if already_existing_one:
+            logger.info("Personal info already exists")
+            return already_existing_one
+        return super()._create_personal_info(entity_data)
```

Comment:
maybe simply `get_or_create()`? ([docs](https://docs.djangoproject.com/en/5.0/ref/models/querysets/#get-or-create))

Reply:
As far as I see from the description, it is for a different use case, this one is simple enough I think

Reply:
Well, alright :) It may not be safe for concurrent requests, but we will have none in these scripts